projects/addon-mobile/components/mobile-dialog/mobile-dialog.tokens.ts

```diff
@@ -0,0 +1,27 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+
+import {TuiMobileDialogOptions} from './mobile-dialog-options';
+
+export type TuiMobileDialogDefaultOptions = Omit<TuiMobileDialogOptions<unknown>, 'data'>;
```

Comment:
I'm not sure we want to export such types. Maybe keep them local? That `data` gives all sorts of headaches for types :D I just don't want to have a bunch of confusing interfaces.